phalcon/forms/form.zep

```diff
@@ -370,10 +370,23 @@ class Form extends Injectable implements \Countable, \Iterator
 	 */
 	public function getMessages(boolean byItemName = false) -> <Group>
```

Comment:
```zep
 	public function getMessages(boolean byItemName = false) -> <Group> | array
```

Reply:
Could you please amend PHPDoc by adding examples from the https://github.com/phalcon/cphalcon/issues/13294 issue